apps/api/src/app/inbound-parse/services/inbound-parse.queue.service.ts

```diff
@@ -34,7 +33,7 @@ export class InboundParseQueueService {
   }
 
   public getWorkerProcessor() {
-    return async ({ data }: { data: InboundEmailParseCommand }) => {
+    return async ({ data }: { data: IInboundParseDataDto }) => {
       Logger.verbose({ data }, 'Processing the inbound parsed email', LOG_CONTEXT);
       await this.emailParseUsecase.execute(InboundEmailParseCommand.create({ ...data }));
```

Comment:
I wonder if we should do the execution of create in this PR or not. it could be a breaking change.